spec/solidus_retail/order/generate_pos_order_spec.rb

```diff
@@ -33,6 +33,36 @@
       subject
       expect(last_order).to be_complete
     end
+
```

Comment:
Then if it's unit test wouldn't be better off to mock everything and maybe use the helper method that I have provided in order to create double object instead of relying in creating Solidus objects?